package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -766,14 +767,18 @@ def _getOneDeltaPoint(self,universe, repInd, i ,t0, dt):
             valHH += self.lg2(np.dot(unitHHVector0,unitHHVectorp))
             valdip +=  self.lg2(np.dot(unitdipVector0,unitdipVectorp))
             n += 1
-        valOH = valOH/n
-        valHH = valHH/n
-        valdip = valdip/n
-        return (valOH,valHH,valdip)
+
+        if n == 0:
+            return (0,0,0)
```

Comment:
Ok, I'll adapt the code to the PEP8 format, but is it necessary to have lines shorter than 80 column even if I lost readability of code?